src/App.js

```diff
@@ -2,15 +2,58 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  };
 
-  add100 = () => {};
+  addOne = () => {
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
+  };
 
-  increase = () => {};
+  add100 = () => {
+    this.setState(state => ({
+      count: state.count + 100,
+    }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0) {
+      this.add100();
+    }
+
+    this.addOne();
+  };
 
   render() {
+    const { count } = this.state;
+
     return (
-      <h1>Count: 0</h1>
+      <>
+        <div className="wrapper">
+          <h1 className="header">
+            Count:
+            {count}
+          </h1>
+
+          <button type="button" className="counter" onClick={this.addOne}>
```

Comment:
```suggestion
          <button
            type="button"
            className="counter"
            onClick={this.addOne}
          >
```
если атрибутов больше одного - переноси их на новую строку каждый